### ISSUE TITLE
[FR]: Column footer

### ISSUE THREAD

**Issue by risalfajar** [bryanmylee/svelte-headless-table#122]
It will be useful to have a footer plugin in which you can create footer for some columns. For example, here we want to create a footer which sums all the values from the columns above it:
![image](https://user-images.githubusercontent.com/28381124/230709255-d4af6a00-6e8a-459d-a45e-7063014c1c6d.png)

My suggested syntax:
```typescript
const columns = table.createColumns([
    table.column({
        header: 'Price',
        accessor: 'price',
        plugins: {
            footer: ({values}) => sum(values)
        }
    }),
])
```

**Comment by risalfajar** [comment]
Even better, we can make it as built-in/non-plugin, just like `header`.

**Comment by risalfajar** [comment]
Just noticed that there's `footer` variable in `Column` class, but can't find it in the docs. What does it do?

**Comment by bryanmylee** [comment]
It was a work in progress, as I couldn't figure out a use-case for it early on, but I'll work on adding it in given the request.

**Comment by risalfajar** [comment]
Thanks, it's very appreciated.

**Comment by emillime** [comment]
I'm also looking for this, I want to add sums at the bottom for some of the columns. Right now it looks like we have to create that manually.

**Comment by AdamAugustinsky** [comment]
Hey guys, currently going through  a similar problem, has anyone been able to make a row with the aggregated data from the other rows in table? Could not find any examples on how to do that.
@bryanmylee

**Comment by risalfajar** [comment]
@AdamAugustinsky You can manually, just add `<tr>` and `<td>` tag in the last row of the table.

**Comment by AdamAugustinsky** [comment]
> @AdamAugustinsky You can manually, just add `<tr>` and `<td>` tag in the last row of the table.

Yes, but more specifically my problem is making an aggregated data row.
How did you end up solving that?

**Comment by risalfajar** [comment]
@AdamAugustinsky I summed the values using Javascript
